### PR TITLE
Texttospeech

### PR DESCRIPTION
**Pull request by gmarkay**

### CODE REVIEW

File: javascripts/main.js [hunk 1]
```diff
@@ -24,7 +24,30 @@ button.addEventListener("click", function() {
...
             break;
     }
+    speak();
+  });
+ 
+
+// function speak(){
+//   let msg = new SpeechSynthesisUtterance(outputDiv.innerText);
+//   let voice = window.speechSynthesis.getVoices();
+//   console.log
```
Comment (electriceeli) [inline javascripts/main.js]: BRO

File: javascripts/main.js [hunk 2]
```diff
@@ -24,7 +24,30 @@ button.addEventListener("click", function() {
...
+// // console.log(speechSynthesis());
```
Comment (electriceeli) [inline javascripts/main.js]: BROOOOOOOO

**Review by electriceeli — CHANGES_REQUESTED** [review]
bro. console logs

done

**Comment by gmarkay** [comment]
removed comments and logs

**Pull request merged**